src/content/resources/index.mdx

```diff
@@ -1,14 +1,15 @@
 ---
 title: Resources
-description: "A full list of resources designed to help you navigate the OASIS+ contract: explore software applications, tools, and documentation."
+description: "A full list of resources designed to help buyers and sellers navigate the OASIS+ contract: explore software applications, tools, and documentation."
 order: 0
 ---
 import TableBorderless from "@components/TableBorderless.astro"
 import TableHeaderBorderless from "@components/TableHeaderBorderless.astro"
 import TableCellBorderless from "@components/TableCellBorderless.astro"
 export const components = {table: TableBorderless, th: TableHeaderBorderless, td: TableCellBorderless}
 
-We have resources to help you navigate the OASIS+ contract; as part of this, we have included descriptions of all software applications, tools, and documentation related to the contract. If you do not see what you need, please reach out to us at [oasisplus@gsa.gov](mailto:oasisplus@gsa.gov).
+We have resources to help buyers and sellers
```

Comment:
Looks like there is some extra space here that's not needed?

Reply:
resolved